app/facades/tagging_spreadsheet_facade.rb

```diff
@@ -0,0 +1,21 @@
+class TaggingSpreadsheetFacade < SimpleDelegator
```

Comment:
There's certainly similarity between facades and presenters. While both are a mechanism for providing a simpler API for underlying objects, I think it's more typical for the latter name to be used when the context is presentational. Specifically - when creating some kind of response at the boundaries of a system for external clients, whether that's forming a JSON payload or rendering ERB. I see the clients of a facade as being internal.

We're deep in bike-shedding territory at this point, but I think the name 'facade' raises more questions than it answers in a system of this size.